VocaDbModel/DataContracts/ReleaseEvents/ReleaseEventSeriesForApiContract.cs

```diff
@@ -29,7 +30,7 @@ public class ReleaseEventSeriesForApiContract {
 			}
 
 			if (fields.HasFlag(ReleaseEventSeriesOptionalFields.Events)) {
-				Events = series.Events.Select(e => new ReleaseEventContract(e, languagePreference)).ToArray();
+				Events = series.Events.Select(e => new ReleaseEventForApiContract(e, languagePreference, ReleaseEventOptionalFields.None, thumbPersister)).ToArray();

```

Comment:
What should I pass as the third argument here? Non for-api data contracts are used in other for-api data contracts (e.g. [SongForApiContract](https://github.com/VocaDB/vocadb/blob/43754bb4b75b5de0d067c775728e8a9bc2d99dc4/VocaDbModel/DataContracts/Songs/SongForApiContract.cs#L106)).

Reply:
None is fine for now. It's easier to add fields than to remove them. That AlbumContract probably was forgotten there. It's better to use the Api contracts.